src/main/resources/api/service_openapi.yaml

```diff
@@ -186,3 +193,17 @@ components:
         application/json:
           schema:
             $ref: '#/components/schemas/ResourceDescription'
+
+  securitySchemes:
```

Comment:
do we need to add `security[something]` in our yaml somewhere?

Reply:
Compared with SAM, Data Repo and WM code, seems they are similar
I think we won't need to add `security: []` for unthenticatevd APIs. For others, and seems we don't need that